src/Compilers/VisualBasic/Portable/BoundTree/Statement.vb

```diff
@@ -29,6 +29,26 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Property
     End Class
 
+    Partial Friend Class BoundAddressOfOperator
```

Comment:
>BoundAddressOfOperator [](start = 25, length = 22)

Both nodes are expressions, consider moving the code to Expression.vb

Reply:
Fixed.